lightning/src/util/message_signing.rs

```diff
@@ -0,0 +1,90 @@
+use crate::util::zbase32;
+use bitcoin::hashes::{sha256d, Hash};
+use bitcoin::secp256k1::recovery::{RecoverableSignature, RecoveryId};
+use bitcoin::secp256k1::{Error, Message, PublicKey, Secp256k1, SecretKey};
+
+static LN_MESSAGE_PREFIX: &[u8] = "Lightning Signed Message:".as_bytes();
```

Comment:
`.as_bytes()` in a const isn't stable in our MSRV, but you can use a `b` prefix.
```suggestion
static LN_MESSAGE_PREFIX: &[u8] = b"Lightning Signed Message:";
```

Reply:
My bad, I actually had it that way but rebased to changed the order of the commits and this must have reset to the old way.